components/ui/label.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
```

Comment:
Only need "use client" if the component is using state or a hook like useEffect, or useMemo. All children components of a client-side component (has "use client" decorator) are also client-side. Having this in a primitive like this label would prevent this from being loaded serverside on a static page.
```suggestion
```